src/components/Disputes/actions/DisputeVoting.js

```diff
@@ -53,41 +53,53 @@ function VotingFinalRound({ draftTermId, onRequestCommit }) {
 
 function VotingActions({ canJurorVote, onRequestCommit }) {
   const wallet = useWallet()
+  const { below } = useViewport()
+  const compactMode = below('medium')
+
+  const buttonWidth = compactMode ? '100% ' : `calc((100% - ${2 * GU}px) /  3)`
 
   return (
     <div>
       <div
         css={`
           display: flex;
           justify-content: space-between;
+          flex-direction: ${compactMode ? 'column' : 'row'};
           width: 100%;
           margin-bottom: ${1.5 * GU}px;
+
+          & > button {
```

Comment:
This is a bit brittle, since `Button` doesn't necessarily render to a `button` element.

It probably still makes sense to make this a custom component, something like:

```js
const VotingButton = styled(Button)`
  width: calc((100% - ${2 * GU}px) / 3);
  margin-bottom: ${({ compactMode }) => `${compactMode ? 1 : 0) * GU}px`)
`
```